types/nightwatch/index.d.ts

```diff
@@ -1854,6 +1854,7 @@ export interface NightwatchAPI
         NightwatchApiCommands {
     baseURL: string;
     assert: NightwatchAssertions;
+    actions(options?: { async: boolean; bridge: boolean } | { async: boolean } | { bridge: boolean }): Actions;
```

Comment:
Had same idea, but not sure, because this would mean you can pass an empty object, which is now prohibited.

If this is allowed, we can choose your solution, of course

Reply:
I was just looking at the selenium code to see if an empty object is allowed and found that `{}` is in fact the [default argument](https://github.com/SeleniumHQ/selenium/blob/1aba14e9c19caff9f5136fbb7d3f7d2836ef07c0/javascript/node/selenium-webdriver/lib/input.js#L596) when [no `options` argument is passed](https://github.com/SeleniumHQ/selenium/blob/1aba14e9c19caff9f5136fbb7d3f7d2836ef07c0/javascript/node/selenium-webdriver/lib/webdriver.js#L789). So, it is safe to include that.

I also tried passing `{}` as an argument in Nightwatch and it works fine.

Reply:
Adapted it